sys/mathoid.js

```diff
@@ -83,7 +84,11 @@ class MathoidService {
                 return P.join(
                     hyper.put({
                         uri: new URI([rp.domain, 'sys', 'key_value', 'mathoid_ng.check', hash]),
-                        headers: checkRes.headers,
+                        headers: {
```

Comment:
Also, since the headers are explicitly set on L79 and then returned, to minimise copy/pasta, let's convert them to `x-store` here in place rather than explicitly list them.

Reply:
done